jobs/openshift4/cluster/create/Jenkinsfile

```diff
@@ -0,0 +1,180 @@
+#!groovy
+// by default build number is used as cluster to prevent clashes with possibly already existing clusters
+String clusterName = BUILD_NUMBER
+String awsRegion = params.AWS_REGION
+String baseDomain = params.DOMAIN_NAME.toString().length() > 0 ? params.DOMAIN_NAME : "cloudservices.rhmi.io"
+if (CLUSTER_NAME) {
+    clusterName = CLUSTER_NAME
+}
+String clusterInfoDirectory = "clusters/" + clusterName
+String installConfigDirectory = ""
+String rootWorkspaceDir = ""
+
+def err = null
+String adminPassword = ""
+
+node("cirhos_rhel7") {
+    timeout(120) { ansiColor("gnome-terminal") { timestamps {
+        try {
+            currentBuild.displayName = "${currentBuild.displayName} ${clusterName}"
+            currentBuild.description = "clusterName: ${clusterName}\nawsRegion: ${awsRegion}"
+            stage("Install openshift-install") {
+                sh(
+                    returnStdout: false,
+                    script: """
+                        wget -q https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-install-linux-4.1.11.tar.gz
+                        tar xvzf openshift-install-linux-4.1.11.tar.gz
+                        ./openshift-install version
+                    """
+                )
+            } // stage
+            
+            stage("Install and configure aws cli") {
+                sh(
+                    returnStdout: false,
+                    script: """
+                        curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
+                        unzip awscli-bundle.zip
+                        sudo ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
+                        /usr/local/bin/aws --version
+                        mkdir ~/.aws
+                        touch ~/.aws/credentials
+                        set +x # suppress output of the next shell command
+                        echo "[default]\naws_access_key_id=${AWS_ACCESS_KEY_ID}\naws_secret_access_key=${AWS_SECRET_ACCESS_KEY}" > ~/.aws/credentials
+                        set -x
+                    """
+                )
+            } // stage
```

Comment:
Any tooling required should be installed inside the agent rather than having to install it every time.

Reply:
good point. I will update the slave once OpenStack and Jenkins are up again